source/katana/updating.md

```diff
@@ -0,0 +1,49 @@
+---
+title: sabre/katana
+product: katana
+layout: default
+---
+
+To update sabre/katana, you have two options.
```

Comment:
The main documented upgrade method should be to unzip the directory and copy over the relevant files. Again, we are assuming people on windows using FTP servers on shared hosting.